xpdschema/schemas/run_start.json

```diff
@@ -3,7 +3,8 @@
   "properties":{
     "facility": {
       "type": "string",
-      "description": "Facility Name (NSLS-II, APS, etc)"
+      "description": "Facility Name (NSLS-II, APS, etc)",
+      "enum": ["NSLS-II", "APS", "LCLS", "ALS", "CHESS"]
```

Comment:
I think that I am going to punt on this for now for two reasons.
1. The probability that we roll this software out to facilities beyond the NSLS-II between now and the next release is small (aka You Aren't Gonna Need It)
2. This may become problematic for TEM and other cases so we should think this through (maybe facility is a bit more complex than we currently have it, eg a dict with various information).